Common/src/Utils/IQueryableExt.cs

```diff
@@ -0,0 +1,58 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2023. All rights reserved.
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+
+namespace ArmoniK.Core.Common.Utils;
+
+public static class IQueryableExt
+{
+  public static IOrderedQueryable<T> OrderByList<T>(this IQueryable<T>                 queryable,
```

Comment:
Do you really have to implement it twice?

Reply:
due to https://github.com/aneoconsulting/ArmoniK.Core/blob/13805d638c8c0162c88070c6649ae53aba8934dc/Adaptors/MongoDB/src/TaskTable.cs#L473
that we created, we cannot change the type of the output of the orderby. For now, I do not have a better solution